Pages/product.html

```diff
@@ -0,0 +1,198 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Casavile Furniture & Accessories</title>
+    <link rel="stylesheet" href="/CSS/product.css">
+
+    <!-- icon link starts here  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+
+    <!-- font link starts here  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Rajdhani:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+
+<body>
+    <nav id="main-nav">
+        <div class="nav-bar">
+            <div class="nav-left">
+                <ul class="list-elements">
+                    <li class="list__item"><img class="logo" src="/img/index-img/logo.png" alt="logo"></li>
+                    <li class="list__item"><a class="nav-link" href="#Furniture">Furniture</a></li>
+                    <li class="list__item"><a class="nav-link" href="#Acceccories">Accessories</a></li>
+                </ul>
+            </div>
+
+
+            <div class="nav-right">
+                <ul class="list-elements">
+                    <li class="list__item"><a class="nav-link" href="login.html" target="_blank"><i class="fa-solid fa-user list__icon"></i> Login</a></li>
+                    <li class="list__item"><a class="nav-link" href="#main-footer"><i class="fa-solid fa-address-card list__icon"></i> Contacts</a></li>
+                    <li class="list__item"><a class="nav-link" href="cart.html" target="_blank"><i class="fa-solid fa-cart-shopping list__icon"></i> Cart</a></li>
```

Comment:
Remove target from anchor links, so user  can stay on the same page and not opening new tabs.